config/ca_cert.yaml

```diff
@@ -4,17 +4,40 @@
 #@ load("@ytt:data", "data")
 #@ load("@ytt:overlay", "overlay")
 
+#@ def split_certs():
+#@   ca_certs_lines=data.values.ca_cert_data.splitlines()
+#@   ca_certs = []
+#@   i1=0
+#@   i2=0
+#@   for ca_cert_line in ca_certs_lines:
+#@     if ca_cert_line == "-----END CERTIFICATE-----" :
```

Comment:
Should we error somehow if the cert does not have `-----END CERTIFICATE-----` in it?

Reply:
I added the validation that the last line of `ca_cert_data` must be `-----END CERTIFICATE-----` but it doesn't feel like a super popular error path